apps/ios/SimpleX Localizations/en.xcloc/Localized Contents/en.xliff

```diff
@@ -531,7 +556,7 @@
       <trans-unit id="Connect" xml:space="preserve">
         <source>Connect</source>
         <target>Connect</target>
-        <note>No comment provided by engineer.</note>
+        <note>server test step</note>
```

Comment:
Maybe create a separate key? "Server test step Connect" and change English translation? At least we should check translations still work, I can do it later